ci/docker/runtime_functions.sh

```diff
@@ -31,14 +31,49 @@ clean_repo() {
     git submodule update --init --recursive
 }
 
+# wrap compiler calls with ccache
+build_ccache_wrappers() {
```

Comment:
do we need wrappers if we compile the latest ccache ourselves? it should have the fix for nvcc right?

Reply:
Yes, a fix has been added as can be seen in [release notes for 3.4](https://github.com/ccache/ccache.git) but this is unfortunately not enough, I couldn't get in working.